compose.yaml

```
@@ -225,4 +225,4 @@ volumes:
   caddy_config:
   caddy_data:
   minio_data:
-  postgres_data:
+  postgres_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)